### PR TITLE
Print names of events and current stage

### PR DESCRIPTION
**Pull request by Joao-Dionisio**
Using ```model.getStage()``` we get an integer, not very informative. We now have ```model.getStageName()```, which gives us the name of the stage. Same for events. This does not solve #766, as directly trying to print an event should still give an error. Similar to PR #484.

Draft because I'm sure there are better ways to do this.

### CODE REVIEW

File: src/pyscipopt/scip.pxi [hunk 1]
```diff
@@ -305,6 +310,19 @@ cdef class Event:
...
+            self._getEventNames()
+        return EventNames[self.getType()]
+
+    def _getEventNames(self):
+        """gets event names"""
+        for name in dir(PY_SCIP_EVENTTYPE):
+            attr = getattr(PY_SCIP_EVENTTYPE, name)
+            if isinstance(attr, int):
+                EventNames[attr] = name
+
```
Comment (mmghannam) [inline src/pyscipopt/scip.pxi:325]: Would be great if you could implement the __str__ method too (read here for more https://www.digitalocean.com/community/tutorials/python-str-repr-functions)

File: src/pyscipopt/scip.pxi [hunk 2]
```diff
@@ -305,6 +310,19 @@ cdef class Event:
...
+    def _getEventNames(self):
+        """gets event names"""
+        for name in dir(PY_SCIP_EVENTTYPE):
+            attr = getattr(PY_SCIP_EVENTTYPE, name)
+            if isinstance(attr, int):
+                EventNames[attr] = name
+        
     def __repr__(self):
         return self.getType()
```
Comment (mmghannam) [inline src/pyscipopt/scip.pxi:327]:
```suggestion
        return str(self.getType())
```
From what I understand __repr__ should always return a string so this behavior is weird.

**Review by mmghannam — CHANGES_REQUESTED** [review]
Thanks João for the useful contribution!

File: src/pyscipopt/scip.pxi (hunk 1 above)
Comment (Joao-Dionisio) [reply src/pyscipopt/scip.pxi:325]: I added it to the Events, Mo, but couldn't do it for the Stages, as they're not from a class.

Comment (mmghannam) [reply src/pyscipopt/scip.pxi:325]: Perfect!

**Review by mmghannam — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,13 +2,7 @@
 
 ## Unreleased
 ### Added
-- Add getConshdlrName to class Constraint
```
Comment (mmghannam) [inline CHANGELOG.md:5]: Something seems to have been merged wrong here, this line should be in the 4.4.0 release, and your change should be in a new _Unreleased_ section.

Comment (Joao-Dionisio) [reply CHANGELOG.md:5]: Oh damn it, I always mess up the CHANGELOG somehow. I think it's alright now. Thanks for looking into this, Mo!

**Pull request merged**